Makefile

```diff
@@ -0,0 +1,89 @@
+###
+# This Makefile provides portable, stack-agnostic commands (unlike npm)
+#
+# These commands are meant to be run from your local machine, not from
+# within the docker container. Each command handles exec'ing into the
+# docker container to run the command
+
+ifneq (,)
+	This makefile requires GNU Make.
+endif
+# #######
+# Basics
+platform-onboarded:
+	make migrate-docker
+	docker exec -it platform_platform_1 /bin/bash
```

Comment:
although most of the times the container will be named platform_platform_1 , it's not guaranteed . If someone decided to rename their "platform" folder to i.e. "bananas" , the container would be named "bananas_platform_1" .

`docker-compose exec platform ...`

would be safer for docker-compose generated containers, as it knows how they have been named